src/explore-education-statistics-frontend/src/components/Page.tsx

```diff
@@ -19,7 +19,7 @@ type Props = {
 
 const Page = ({
   title,
-  caption = 'Find, download and explore official Department for Education (DfE) statistics and data in England.',
+  caption = '',
```

Comment:
I think it the caption has been blanked out deliberately here?

Reply:
Yes it's been done deliberately